README.md

```diff
@@ -36,8 +36,13 @@ Because a user's folder only holds items that they own, it effectively checks al
 1. Clone the repository
    * `cd <my git directory>`
    * `git clone https://github.com/agrc/auditor.git`
-1. Create a `credentials.py` file in the repo's directory using `credentials_template.py`.
+1. Create a `credentials.py` file in the auditor directory using `credentials_template.py`.
    * DO NOT check `credentials.py` into version control! The repo's `.gitignore` has been set to ignore `credentials.py`; verify this on your local repo.
+1. Install the supervisor prerequisite: (someday this may be pypi'd)
+   * `cd <my git directory>`
+   * `git clone https://github.com/agrc/supervisor.git`
+   * `cd <my git directory>\supervisor`
+   * `pip install .`
```

Comment:
I think that you could get this pushed to pypi in about the time it takes for someone to follow these steps. Check this doc out: https://github.com/agrc/sweeper#uploading-to-pypiorg

Reply:
Assuming I didn't need to get an account set up and learn just what I was doing with PyPi first ;)

It's been uploaded as agrc-supervisor. Change to docs forthcoming.